tests/integration/tier1/one-kernel-scenario/run_conversion.py

```diff
@@ -1,17 +1,14 @@
 import os
-import platform
 
 
-def test_run_conversion_using_custom_repos(shell, convert2rhel):
+def test_run_conversion_using_custom_repos(shell, convert2rhel, get_system_release):
     """TODO better description and function name"""
```

Comment:
Not related to the scope of the PR, but you could add 
```suggestion
    """TODO(r0x0d) better description and function name"""
```
here and I will take care of that later. Just checked the blame and I added that line